### ISSUE TITLE
Users blocked by password reset & beta webmaker redirect

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/id.webmaker.org#335]
We recently started forwarding traffic from the webmaker.org landing page to beta.webmaker.org. This redirect interferes with anyone who needs to add/reset their password to recover their projects. This affects more than a few people because we are sunsetting tools and Persona users don't have a password. 

Questions
 1. Is this a critical issue or a small hassle? 
 1. If so, should we suspend the redirect until we have a fix?
 1. When we archive webmaker to old.webmaker.org in the near future, will we encounter the same issue? @cade

Note
 - Users can add/reset a password via teach.mozilla.org and then login to Webmaker

cc @hpauljohnson @hannahkane @ldecoursy @amirad @KevZawacki @jbuck @simonwex

**Comment by xmatthewx** [comment]
Technical question @cadecairos: 

Reset links look like:
 - via Webmaker: https://webmaker.org/?uid=...
 - via Teach: https://id.webmaker.org/reset-password?client_id=...

Can we just update Webmaker to utilize id dot webmaker like teach?

**Comment by cadecairos** [comment]
we can do that, but that won't fix events.wmo, login.wmo, popcorn.wmo, thimble.wmo and goggles.wmo (possibly other that I forgot) since they all need individual patches to set that up.

**Comment by xmatthewx** [comment]
Link in reset email now uses the path /explore to avoid the redirect. I tested on Teach, Webmaker Android, and Webmaker.org. No problems there. I hit rate limiting on Popcorn. Seems resolved. Will test with a few others.
